src/util/classes.ts

```diff
@@ -2,35 +2,38 @@ const _ = require('lodash');
 
 const level_sep = ">";
 
+interface Rule {
+    type: string;
+    regex?: string;
+    ignore_case?: boolean;
+}
+
 interface Category {
     id?: number;
     name: string[];
     name_pretty?: string;
     subname?: string;
-    rule: {
-        type: string;
-        pattern: string;
-    };
+    rule: Rule;
     depth?: number;
     parent?: string[];
     children?: Category[];
 }
 
 export const defaultCategories: Category[] = [
-  { name: ['Work'], rule: { type: 'regex', pattern: 'Google Docs' } },
+  { name: ['Work'], rule: { type: 'regex', regex: 'Google Docs' } },
   {
     name: ['Work', 'Programming'],
-    rule: { type: 'regex', pattern: 'GitHub|Stack Overflow' },
+    rule: { type: 'regex', regex: 'GitHub|Stack Overflow' },
   },
   {
     name: ['Work', 'Programming', 'ActivityWatch'],
-    rule: { type: 'regex', pattern: '[Aa]ctivity[Ww]atch|aw-' },
+    rule: { type: 'regex', regex: 'ActivityWatch|aw-', ignore_case: true },
```

Comment:
This ignore_case breaks the query on my machine as it's directly converted to a string and the aw-core expects "True" while this is sent as a "true".